gulp-ng-config.js

```diff
@@ -67,10 +68,12 @@ function gulpNgConfig (moduleName, configuration) {
 
     jsonObj = _.merge({}, jsonObj, configuration.constants || {});
 
+    var spaces = (configuration.pretty) ?  2 : 0;
```

Comment:
Perhaps we allow the configuration to specify how many spaces? I could see the input logic being:

- `falsy` :arrow_right: `0`
- `true` :arrow_right: `2` (default)
- `Integer` :arrow_right: use the supplied value